tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -0,0 +1,25 @@
+*** Settings ***
+Resource    ../../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource    ../../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+
+
+*** Test Cases ***
+Verify Pytorch And Tensorflow Can Be Spawned
+    [Documentation]    Verify Pytorch and tensorflow can be spawned
+    [Tags]    Sanity
+    ...       RHODS-1469
+    Verify Cuda Builds Are Completed
+    Verify Image Can Be Spawned  image=pytorch  size=Default
+    Verify Image Can Be Spawned  image=tensorflow  size=Default
```

Comment:
This might work because you are calling `Begin Web Test` twice (and opening two browsers), but you're never closing either the browser or the jupyterlab session of the user (which might get closed by `Fix Spawner Status` in `Begin Web Test` only for the first one).

You should either modify the `Verify Image Can Be Spawned` keyword to also close the session and the browser, or you should add those steps here in the TC.

Reply:
Added End Web Test in "Verify Image Can Be Spawned"